jest.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  // Indicates whether the coverage information should be collected while executing the test
```

Comment:
Do we need these comments? They will be carried with every project. This is the role of documentation, not a config to explain fields.